galaxy_importer/constants.py

```diff
@@ -44,12 +44,22 @@
 MAX_TAGS_COUNT = 20
 NAME_REGEXP = re.compile(r"^(?!.*__)[a-z]+[0-9a-z_]*$")
 
+# For these extensions we support listing them in the galaxy contents list
+# In the future we may allow any extension to be listed, and call ansible-doc on it
+EDA_EVENT_SOURCE_NAME = "eda/plugins/event_sources"
+EDA_EVENT_FILTER_NAME = "eda/plugins/event_filters"
```

Comment:
These may need to be updated to:
EDA_EVENT_SOURCE_NAME = "eda/plugins/event_source"
EDA_EVENT_FILTER_NAME = "eda/plugins/event_filter"

Reply:
@benthomasson

Reply:
It is singular